### PR TITLE
Add owning org filter to schemes

### PR DESCRIPTION
**Pull request by ghost**
Add owning org filter to Schemes
https://dluhcdigital.atlassian.net/browse/CLDC-2453

### CODE REVIEW

File: app/services/filter_manager.rb [hunk 1]
```diff
@@ -72,7 +73,8 @@ def session_filters(specific_org: false)
   def deserialize_filters_from_session(specific_org)
     current_filters = session[session_name_for(filter_type)]
     new_filters = current_filters.present? ? JSON.parse(current_filters) : {}
-    if @filter_type.include?("logs")
+
+    if filter_type.include?("logs")
```
Comment (kosiakkatrina) [inline app/services/filter_manager.rb:77]: Should this still be @filter_type?

Comment (ghost) [reply app/services/filter_manager.rb:77]: Is there any specific reason I am not seeing? there's an `attr_reader` for this variable at the top of the file

Comment (kosiakkatrina) [reply app/services/filter_manager.rb:77]: oh right 🤦‍♀️ I was so confused about where you're getting it from.. All good

**Review by kosiakkatrina — APPROVED** [review]

**Review by natdeanlewissoftwire — APPROVED** [review]
lgtm!

**Pull request merged**